### PR TITLE
Remove old intellisense decorations after changing languages

### PR DESCRIPTION
**Pull request by lucyzhang929**
This PR fixes #https://github.com/microsoft/azuredatastudio/issues/11689

### CODE REVIEW

File: src/vs/editor/common/services/modelServiceImpl.ts [hunk 1]
```diff
@@ -617,6 +617,7 @@ export class ModelServiceImpl extends Disposable implements IModelService {
 	}
 
 	private _onDidChangeLanguage(model: ITextModel, e: IModelLanguageChangedEvent): void {
+		model.removeAllDecorationsWithOwnerId(0);
```
Comment (Charles-Gagnon) [inline src/vs/editor/common/services/modelServiceImpl.ts:620]: What's owner ID 0 refer to?

Comment (Charles-Gagnon) [inline src/vs/editor/common/services/modelServiceImpl.ts:620]: So why is this required for our editors? This issue doesn't seem to happen for VS Code editors so it'd be good to find out why that is before we start changing their code.

(and FYI - in the future if we ever make changes to code under the vs folder you should be adding a `{{ SQL CARBON EDIT }}` comment to the lines you're changing so we can track that when doing merges.

Comment (kburtram) [inline src/vs/editor/common/services/modelServiceImpl.ts:620]: please add `// {{ SQL CARBON EDIT }}` to all changes under `vs` prior to merging.  I'm also unclear why this change is required.

**Comment by lucyzhang929** [comment]
Sorry - didn't realize I was making changes in vs. I think the call to remove decorations should be made elsewhere. I'll update this PR once I've made the changes. Thanks!
